api/controllers/linux/install/infra.go

```diff
@@ -67,7 +67,16 @@ func (c *InstallController) SetupInfra(ctx context.Context, ignoreHostPreflights
 			}
 		}()
 
-		if err := c.infraManager.Install(ctx, c.rc); err != nil {
+		configValues := make(map[string]string)
+		if c.store != nil && c.store.AppConfigStore() != nil {
```

Comment:
nit: no need to check if stores are nil, nothing would function if that was true.